python/rikai/contrib/tfhub/tensorflow/ssd.py

```diff
@@ -37,27 +36,33 @@ def transform(self) -> Callable:
         return None
 
     def predict(self, images, *args, **kwargs) -> Any:
+        print("images shape", images.shape)
+        # one image should have shape (height,width,3),
```

Comment:
This model does not support batching as stated here:

https://tfhub.dev/tensorflow/ssd_mobilenet_v2/2

> A three-channel image of variable size - the model does NOT support batching. The input tensor is a tf.uint8 tensor with shape [1, height, width, 3] with values in [0, 255].

I suggest that you should add some comments here.

Reply:
updated, do you have any new ideas here? https://github.com/eto-ai/rikai/pull/574/files#diff-0ee92fc67483ddd537672653ff62e0da925334688d1a92cc5c1166d5dec840f7R76